### ISSUE TITLE
The XML tags aren't getting converted properly into HTML tags

### ISSUE THREAD

**Issue by jaydeept** [domaindrivendev/Swashbuckle.AspNetCore#530]
I am migrating my swagger web api page from .net framework to .net core 2. In the older page, I was using SwaggerResponse with error code and as I had same status code for two errors, I was using <p> tag hence description of error will be displayed in two lines.
In .net core 2, it's printing <p> as it is and not able to parse it properly. This seems to be issue with NuGet package Swashbuckle.AspNetCore.SwaggerGen.
The XML tags aren't getting converted properly into HTML tags I believe so.

`[SwaggerResponse(400, typeof(string), "<p>" + WebApiErrorMessages.EmptyQueryText
                                              + "</p><p>" + WebApiErrorMessages.InvalidQuerySyntax
                                              + "</p>")]`

**Comment by funkycoding** [comment]
You can use something like this:

```
/// <summary>
/// Returns an order.
/// </summary>
/// <param name="ordernumber">The ordernumber.</param>
/// <response code="404">Not found. More descriptive text here.</response>
/// <response code="500">Server error. More descriptive text here.</response>
[HttpGet("{ordernumber}")]
[ProducesResponseType(typeof(Order), (int)HttpStatusCode.OK)]
[ProducesResponseType(typeof(string), (int)HttpStatusCode.NotFound)]
[ProducesResponseType(typeof(string), (int)HttpStatusCode.InternalServerError)]
[Produces("application/json")]
public async Task<IActionResult> GetOrderByOrderNumberAsync([FromRoute] string ordernumber)
{
     // code here
}
```

You need to enable document generation in your csproj file:
```
<PropertyGroup>
  <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\ApplicationNameHere.xml</DocumentationFile>
</PropertyGroup>
```

and in your startup (ConfigureServices) you can then link the xml docs to the generated swagger docs:
```
services.AddSwaggerGen(config =>
{
    config.SwaggerDoc("v1", new Info { Title = SWAGGER_API_NAME, Version = "v1" });

    var docFile = $"{Assembly.GetEntryAssembly().GetName().Name}.xml";
    var filePath = Path.Combine(AppContext.BaseDirectory, docFile);

    if (File.Exists((filePath)))
    {
        config.IncludeXmlComments(filePath);
    }
});
```

**Comment by jaydeept** [comment]
I am aware of ProducesResponseType but with that also I am unable to provide paragraph in description using tag <p> or <para>.
I am facing similar issue for <code> tag. I believe, though XML file contains valid tags, those aren't rendering properly in HTML page as HTML tags aren't getting created for few XML tags.

**Comment by funkycoding** [comment]
@jaydeept i think this is a swagger-ui 2.x issue...but they haven't fixed this in 2.x. 
https://github.com/swagger-api/swagger-ui/issues/1622
-or here-
https://github.com/swagger-api/swagger-ui/pull/1632

i was actually trying out swagger-ui 3.4.4 on a local branch, and it seems to be working there (both ways, either via SwaggerResponseAttribute or xml docs). 

![mardown-sw](https://user-images.githubusercontent.com/307410/32954470-7c355e50-cbb3-11e7-893a-eccf585fb2d8.png)

**Comment by funkycoding** [comment]
the swagger-ui is embedded within the swashbuckle package, and that is currently running swagger-ui version 2.2.10. See here: https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/master/src/Swashbuckle.AspNetCore.SwaggerUI/bower.json

You can customize / replace the index.html that's being used by swashbuckle. See https://github.com/domaindrivendev/Swashbuckle.AspNetCore/tree/master/test/WebSites/CustomIndexHtml for an example of a custom index.html. You can get the lastest dist of swagger-ui here: https://github.com/swagger-api/swagger-ui/tree/master/dist. I haven't tried it but it should be possible to use that and the accompanying index.html

**Comment by jaydeept** [comment]
Yes I was able to do that. 
May I know when Swashbuckle package will be updated with new version of swagger-ui?

**Comment by domaindrivendev** [comment]
The new Swagger UI (i.e. 3.x) is available in Swashbuckle version 2.0.0 and above. Please upgrade to the latest version. If you're issue still persists, feel free to re-open
